cmd/trtl/main.go

```diff
@@ -268,8 +321,37 @@ func main() {
 
 // serve starts the trtl server and blocks until it is stopped.
 func serve(c *cli.Context) (err error) {
-	// TODO: invoke replica server
+	var conf config.Config
+	var db *honu.DB
+
+	replicaConfig := honuconfig.ReplicaConfig{
+		PID:    c.Uint64("pid"),
+		Region: c.String("region"),
+	}
+	if db, err = honu.Open(c.String("db"), replicaConfig); err != nil {
+		return cli.Exit(err, 1)
+	}
```

Comment:
Could we please move this to `honu.New` -- all we want the CLI to only deal with configuration and execution, it shouldn't be responsible for setup, initialization, or runtime.